staging/src/k8s.io/apimachinery/pkg/test/apis_meta_v1_unstructed_unstructure_test.go

```diff
@@ -458,21 +458,19 @@ func TestDecodeNumbers(t *testing.T) {
 		}
 	}`)
 
-	pod := &api.Pod{}
+	pod := &example.Pod{}
 
-	// Decode with structured codec
-	codec, err := testapi.GetCodecForObject(pod)
-	if err != nil {
-		t.Fatalf("unexpected error: %v", err)
-	}
-	err = runtime.DecodeInto(codec, originalJSON, pod)
+	_, codecs := TestScheme()
+	codec := apitesting.TestCodec(codecs, schema.GroupVersion{Group: "", Version: runtime.APIVersionInternal})
+
+	err := runtime.DecodeInto(codec, originalJSON, pod)
 	if err != nil {
 		t.Fatalf("unexpected error: %v", err)
 	}
-	// ensure pod is valid
-	if errs := validation.ValidatePod(pod); len(errs) > 0 {
-		t.Fatalf("pod should be valid: %v", errs)
-	}
+	//ensure pod is valid
+	//if errs := validation.ValidatePod(pod); len(errs) > 0 {
```

Comment:
@deads2k could we omit those validation here ? validation is only for internal type right ?